recipe/meta.yaml

```diff
@@ -23,6 +23,8 @@ build:
   number: {{ number }}
   skip: true  # [win or py<37]
   skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
+  skip: true  # [libabseil != '20220623.0']
+  skip: true  # [grpc_cpp != '1.47']
```

Comment:
These skips will be removable once grpc is reduced to one entry in the global pinning: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3399